cypress/e2e/data-import/settings/delete-an-existing-match-profile.cy.js

```diff
@@ -0,0 +1,99 @@
+import getRandomPostfix from '../../../support/utils/stringTools';
+import getRandomStringCode from '../../../support/utils/genereteTextCode';
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { EXISTING_RECORDS_NAMES } from '../../../support/constants';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import ActionProfiles from '../../../support/fragments/data_import/action_profiles/actionProfiles';
+import Users from '../../../support/fragments/users/users';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import FieldMappingProfileView from '../../../support/fragments/data_import/mapping_profiles/fieldMappingProfileView';
+import NewActionProfile from '../../../support/fragments/data_import/action_profiles/newActionProfile';
+import NewFieldMappingProfile from '../../../support/fragments/data_import/mapping_profiles/newFieldMappingProfile';
+import NewJobProfile from '../../../support/fragments/data_import/job_profiles/newJobProfile';
+import NewMatchProfile from '../../../support/fragments/data_import/match_profiles/newMatchProfile';
+import MatchProfiles from '../../../support/fragments/data_import/match_profiles/matchProfiles';
+import MatchProfileView from '../../../support/fragments/data_import/match_profiles/matchProfileView';
+import ConfirmDelete from '../../../support/fragments/data_import/match_profiles/modals/confirmDelete';
+import ExceptionDelete from '../../../support/fragments/data_import/match_profiles/modals/exceptionDelete';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const matchProfileToDelete = {
+      profileName: `C2346 autotest action profile to delete ${getRandomStringCode(8)}`,
+      incomingRecordFields: {
+        field: '001',
+      },
+      existingRecordFields: {
+        field: '001',
+      },
+      matchCriterion: 'Exactly matches',
+      existingRecordType: EXISTING_RECORDS_NAMES.MARC_BIBLIOGRAPHIC,
+    };
+    const profile = {
+      createJobProfile: `autotest jobProfileForCreate.${getRandomPostfix()}`,
+      createActionProfile: `autotest actionProfileForCreate${getRandomPostfix()}`,
+      createMappingProfile: `autotest mappingProfileForCreate${getRandomPostfix()}`,
+      createMatchProfile: `autotest matchProfileForCreate${getRandomPostfix()}`,
+    };
+
+    const calloutMessage = `The match profile "${matchProfileToDelete.profileName}" was successfully deleted`;
+
+    before('Create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        NewFieldMappingProfile.createMappingProfileViaApi(profile.createMappingProfile).then(
```

Comment:
You don't need to create mapping and action profiles. It's enough only match and job profile